src/tournament/tournament.module.ts

```diff
@@ -0,0 +1,18 @@
+import { Module } from '@nestjs/common';
+import { TournamentController } from './controllers/tournament.controller';
+import { TournamentService } from './Services/Tournament.service';
+import { SequelizeModule } from '@nestjs/sequelize';
+import { Tournament } from './models/tournament.model';
+// import { ConfigService } from '@nestjs/config';
+
+@Module({
+  imports: [SequelizeModule.forFeature([Tournament])],
+  controllers: [TournamentController],
+  providers: [
+    {
+      provide: 'ITournamentService',
+      useClass: TournamentService,
+    },
+  ],
```

Comment:
O uso de interface aqui não é necessário. Você não está trabalhando com outras implementações e pelo funcionamento do NestJS referenciar diretamente a implementação não vai te impedir de _mockar_.

```suggestion
  providers: [TournamentService],
```